react-intervaler/src/App.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import './App.scss';
+import './stylesheets/index.scss';
+import Stopwatch from './components/Stopwatch'
+import About from './components/About'
+import Timer from './components/Timer'
+import Error from './components/Error';
+import Navigation from './components/Navigation';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Use styled components instead of importing stylesheets

https://bootstrap-styled.github.io/bootstrap-styled/

Reply:
Bootstrap styled is not supported with typescript

Reply:
I will style components a different way